src/main/resources/licenses/licenses-transform.xsl

```diff
@@ -208,7 +208,7 @@
         </xsl:call-template>
       </xsl:when>
       <!-- MIT -->
-      <xsl:when test="contains(url/text(), 'www.opensource.org/licenses/mit-license') or contains(url/text(), 'www.opensource.org/licenses/MIT') or contains(url/text(), 'jsoup.org/license')">
+      <xsl:when test="contains(url/text(), 'www.opensource.org/licenses/mit-license') or contains(url/text(), 'www.opensource.org/licenses/MIT') or contains(url/text(), 'jsoup.org/license') or contains(url/text(), 'http://opensource.org/licenses/MIT')">
```

Comment:
Seems all the conditions omit the `http://` scheme. Maybe we should just remove the `www.` prefix from one of the existing conditions -- then it would match for your URL too.

Reply:
Np, good optimization...